src/Parser/Hunter/Marksmanship/Modules/Items/MKIIGyroscopicStabilizer.js

```diff
@@ -12,14 +12,23 @@ class MKIIGyroscopicStabilizer extends Module {
     combatants: Combatants,
   };
 
+  usedBuffs = 0;
+
   on_initialized() {
     this.active = this.combatants.selected.hasHands(ITEMS.MKII_GYROSCOPIC_STABILIZER.id);
   }
+  on_byPlayer_removebuff(event) {
+    const buffId = event.ability.guid;
+    if (buffId !== SPELLS.GYROSCOPIC_STABILIZATION.id) {
+      return;
+    }
+    this.usedBuffs += 1;
+  }
```

Comment:
I suspect this could happen with *expiring* the buff and not using it as well (although highly unlikely since you like to cast Aimed Shots). I guess this module will get another update soon so I'll leave that be for now but consider looking into this a little bit more.

Reply:
The buff has no duration, so it can't expire 😄

Reply:
Didn't know that, then it's fine :D